Makefile

```diff
@@ -1,8 +1,23 @@
+###########################
+# Configuration Variables #
+###########################
 ORG := github.com/operator-framework
 PKG := $(ORG)/combo
 VERSION_PATH := $(PKG)/pkg/version
 GIT_COMMIT := $(shell git rev-parse HEAD)
 DEFAULT_VERSION := v0.0.1
+CONTROLLER_GEN=$(Q)go run sigs.k8s.io/controller-tools/cmd/controller-gen
+GO_BUILD := $(Q)go build
+PKGS = $(shell go list ./...)
+
+# Binary build options
+KUBERNETES_VERSION=v0.22.2
+GO_OS=
```

Comment:
is the right side of this statement missing?

Reply:
Ah, looks like some left over code. Deleting.